helm/hpcc/values.schema.json

```diff
@@ -1430,6 +1430,9 @@
         },
         "egress": {
           "$ref" : "#/definitions/egress"
+        },
+        "preferredDataReadPlanes": {
```

Comment:
Should this also be added to roxie?

Reply:
hm it was supposed to be, will add.
(I tested under hthor, thor and roxie-workunit)